examples/make-test-c.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Why not just a Makefile?

It might be probably even better if this uses the **installed** version of SRT. For example, instruct the user that they should do:
1. Configure with installation directory of your choice, may be also inside this directory, e.g. "installed": `configure --prefix=installed`
2. make && make install
3. Compile this pointing the directory in SRTDIR variable: `make SRTDIR=../installed`
4. Makefile will contain something like:
```
FLAGS=$(shell PKG_CONFIG_PATH=$(SRTDIR)/lib/pkgconfig pkg-config --cflags --libs srt openssl)

test-c-server: test-c-server.c
test-c-client: test-c-client.c
%:%.c
    $(CC) -o $@ $^ $(FLAGS) -lstdc++
```

Reply:
Hi @ethouris ,

I know about your opinion. And you're right!
However, I think on this examples as a "guide". This it's the reason for a very simple script. So, I suggest to commit in this form (at time).

Regarding the "install" I suggest to support two ways in the "configure": Option 1 it's the "make install". And the Option 2 it's "make package", that generates a target directory with "target/lib", "target/include" and some examples. Why this? For use it for cross-compilation *without* installing it.

Reply:
This is because this can also demonstrate to the users how they can write their applications. In order to have a directory "target" with installed things, it's enough that you simply do `configure --prefix=target`. Although I think there could be a script that would first compile the SRT library into a directory local to `examples` and then continues with compiling and linking against it...